src/Symfony/Component/Form/Extension/Validator/Event/PostValidateEvent.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Form\Extension\Validator\Event;
+
+use Symfony\Component\Form\FormEvent;
+
+/**
+ * This event is dispatched after validation completes.
+ *
+ * In this stage, the form will return a correct value to Form::isValid() and allow for
+ * further working with the form data.
+ */
+final class PostValidateEvent extends FormEvent
```

Comment:
just asking, why not 

```suggestion
final class PostValidationEvent extends FormEvent
```

?
same for `PreValidateEvent` and `PreValidationEvent`

Reply:
You've got a point. I'm not entirely convinced by "validate" either, but I'm open for ideas. Any opinions or objections to the suggested wording?

Reply:
first i see the listener is called validationListener, thus suggest using validationEvent
but then, i saw the method is called validateForm, so maybe the word validate is ok

(i have in mind the workflow event logic, where event are named after the method) :)